packages/lib/src/auth/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Auth } from 'aws-amplify';
+
+import { createAuthorizationHook } from './auth.hook';
+
+const getAmplifyJwt = async () => {
+  const session = await Auth.currentSession();
```

Comment:
I was wondering about the performance of Auth.currentSession() and I see the question has been asked online (https://www.repost.aws/questions/QUNJUlUrpDTQithn9niYFHaA/how-to-efficiently-retrieve-tokens-from-cognito-while-using-amplify-with-react-native)

This is OK for now but in the future we may want to come back and apply caching here - https://docs.amplify.aws/lib/auth/advanced/q/platform/js/#retrieve-jwt-tokens - to avoid a performance hit on each request.

Reply:
Great find. Saving this!